tests/conftest.py

```diff
@@ -752,6 +751,32 @@ def page(db):
     return page
 
 
+@pytest.fixture
+def page_list(db):
+    data_1 = {
+        'slug': 'test-url',
+        'title': 'Test page',
+        'content': 'test content'}
+    data_2 = {
+        'slug': 'test-url-2',
+        'title': 'Test page',
+        'content': 'test content'}
+    page_1 = Page.objects.create(**data_1)
+    page_2 = Page.objects.create(**data_2)
+    return page_1, page_2
+
+
+@pytest.fixture
+def page_list_unpublished(db):
+    page_1 = Page.objects.create(slug='page-1', title='Page 1',
```

Comment:
Can you do `create` with `bulk_create`? https://docs.djangoproject.com/en/dev/ref/models/querysets/#bulk-create
Less db queries, the same result.